src/tests/unit/tests/popup/components/file-url-unsupported-message-panel.test.tsx

```diff
@@ -35,4 +27,28 @@ describe('FileUrlUnsupportedMessagePanel', () => {
 
         expect(wrapper.getElement()).toMatchSnapshot();
     });
+
+    it('has a NewTabLink that uses createTab to open the manage extension page', () => {
+        const stubExtensionPageUrl = 'protocol://extension-page';
+        const browserAdapterMock = Mock.ofType<BrowserAdapter>(null, MockBehavior.Strict);
+        browserAdapterMock.setup(adapter => adapter.createTab(stubExtensionPageUrl)).verifiable(Times.once());
+
+        // It's important that we use createTab onClick for this link rather than using href,
```

Comment:
not sure how this comment here is relevant to the line it presides...maybe if you move it to the class itself it make more sense.

Reply:
moved into impl by the onClick specification